### PR TITLE
Updated install instructions with alternate location of py4j jar file

### PR DESCRIPTION
**Pull request by varunagrawal**
py4j jar file installs at `/usr/local/share/py4j/` on my system (Ubuntu 14.04) when I use pip install.
Updated install.rst to reflect that.

### CODE REVIEW

File: py4j-web/install.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ Using easy_install or pip
    location depends on the platform and the installation type. Some likely
    locations are:
 
-   1. ``/usr/share/py4j/py4j0.x.jar`` for system-wide install on Linux.
+   1. ``/usr/share/py4j/py4j0.x.jar`` for system-wide install on Linux. Alternatively, it can be at ``/usr/local/share/py4j/py4j0.x.jar``.
```
Comment (bartdag) [inline py4j-web/install.rst]: How about:

``/usr/share/py4j/py4j0.x.jar`` or ``/usr/local/share/py4j/py4j0.x.jar`` for system-wide install on Linux.

**Comment by bartdag** [comment]
Thanks, I would rephrase this just a bit, and I'll merge it :-)

**Comment by varunagrawal** [comment]
@bartdag done!

**Comment by varunagrawal** [comment]
Alright, seems the Travis build is passing as well this time. We are good to merge. :smile:

**Comment by bartdag** [comment]
Thanks for your help!
